homeassistant/components/switch/wemo.py

```diff
@@ -22,7 +22,10 @@ def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     import pywemo.discovery as discovery
 
     if discovery_info is not None:
-        device = discovery.device_from_description(discovery_info[2])
+        location = discovery_info[2]
+        mac = discovery_info[3]
+        # pylint: disable=too-many-function-args
```

Comment:
Don't disable this. It's an actual bug because you try to call something with two args that requires 1. This is happening because you didn't update requirements_all.txt so the old pywemo is still loaded